### DISCUSSION TITLE
General - Chinese Only

### DISCUSSION THREAD

**Discussion by zsr-lukezhang** [zsr-lukezhang/TranslateUI#2] [General]
Chat here!
Talk about anything! 😆

**Comment by zsr-lukezhang** [comment]
调试的时候偶然发现的...
《美式中文》
![image](https://github.com/user-attachments/assets/8d4e7b44-9108-4f49-8acf-f9d35e2e039c)

**Reply by lingbopro** [reply]
这巨硬翻译老特性了，`English`后面但凡跟个东西就会变`中文`

应该是为了方便英文网站的翻译

**Reply by zsr-lukezhang** [reply]
对了
你能不能写个翻译时禁用并显示 `ProcessRing`

**Reply by zsr-lukezhang** [reply]
emm
`ProcessBar` 也许更好看一点
`XAML`我来写吧

**Reply by zsr-lukezhang** [reply]
o 是 `Progress` 不是 `Process`

**Comment by zsr-lukezhang** [comment]
@lingbopro 看看新版本

**Reply by lingbopro** [reply]
看了，挺好

**Reply by zsr-lukezhang** [reply]
你看的是哪个

**Reply by lingbopro** [reply]
都看了

**Reply by zsr-lukezhang** [reply]
`v1.1.2`?

**Reply by lingbopro** [reply]
> `v1.1.2`?

嗯，新的几个我都看了

（因为我关注了这个仓库）
